lib/data/dev_profile_fields.csv

```diff
@@ -1,18 +1,18 @@
 Display email on profile,check_box,,,Basic,settings_only
-Website URL,text_field,https://yoursite.com,,Basic,header
-Summary,text_area,A short bio...,,Basic,header
+Website URL,text_field,https://yoursite.com,,Basic,settings_only
```

Comment:
In this file, I've moved all of the fields that shouldn't be handled by the code rendering "header" and "left_sidebar" fields into the "settings_only" group. I think some of this should be revisited and we might want to add a group such as "links" and another called "protected."

Protected fields should not be deletable or heavily modifiable - in fact, we might not want to render them in the admin UI at all.

Reply:
> In this file, I've moved all of the fields that shouldn't be handled by the code rendering "header" and "left_sidebar" fields into the "settings_only" group.

We should consider writing a data update script adding a "Links" group and moving links there, then update this accordingly. I think it's a small enough change to be part of this PR, but feel free to do it in a follow-up if you prefer that.

> Protected fields should not be deletable or heavily modifiable

Now that name moved back to `User`, what's an example of a protected field? If it can't be deleted or modified, maybe it shouldn't be a profile field at all.

Reply:
@citizen428 I think you are correct in that some things shouldn't be profile fields at all. For context, I was thinking of `summary`specifically when I wrote this thought down.